website/docs/r/maps_account.html.markdown

```diff
@@ -50,7 +50,7 @@ The following arguments are supported:
 
 * `local_authentication_enabled` - (Optional) Is local authentication enabled for this Azure Maps Account? When `false`, all authentication to the Azure Maps data-plane REST API is disabled, except Azure AD authentication. Defaults to `true`.
 
-* `location` - (Optional) The Location in which the Azure Maps Account should be provisioned. Changing this forces a new resource to be created. Defaults to `global`.
+* `location` - (Required) The Location in which the Azure Maps Account should be provisioned. Changing this forces a new resource to be created.
```

Comment:
Could I ask you to scoot the position of this up so that it is listed between `resource_group_name` and `sku_name`? `Required` properties are grouped together by convention in the documentation.